src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -98,6 +109,33 @@ internal static class ParameterHelpers
             return parameters;
         }
 
+        private static bool IsPropertyWithoutBody(Symbol owner)
+        {
+            if (owner is SourcePropertySymbol)
```

Comment:
> So that we can see if it has a non-null body syntax for the get/set methods.

Would there be a compiler error if you replace SourcePropertySymbol with PropertySymbol?